internal/pkg/levee/testdata/src/example.com/tests/select/tests.go

```diff
@@ -122,3 +122,63 @@ func TestTaintedInForkedCall(objects chan interface{}) {
 func PutSource(objects chan<- interface{}) {
 	objects <- core.Source{}
 }
+
+func TestRecvFromTaintedAndNonTaintedChans(sources <-chan *core.Source, innocs <-chan *core.Innocuous) {
+	select {
+	case s := <-sources:
+		core.Sink(sources) // want "a source has reached a sink"
+		core.Sink(s)       // want "a source has reached a sink"
+	case i := <-innocs:
+		core.Sink(innocs)
+		core.Sink(i)
+	}
+	core.Sink(sources) // want "a source has reached a sink"
+	core.Sink(innocs)
+}
+
+func TestSendOnTaintedAndNonTaintedChans(i1 chan<- interface{}, i2 chan<- interface{}) {
+	select {
+	case i1 <- core.Source{}:
+		core.Sink(i1) // want "a source has reached a sink"
+	case i2 <- core.Innocuous{}:
+		core.Sink(i2)
+	}
+	core.Sink(i1) // want "a source has reached a sink"
+	core.Sink(i2)
+}
+
+func TestDaisyChain(srcs chan core.Source, i1, i2, i3, i4 chan interface{}) {
+	select {
+	case s := <-srcs:
+		i1 <- s
+	case z := <-i1:
+		i2 <- z
+	case y := <-i2:
+		i3 <- y
+	case x := <-i3:
+		i4 <- x
+	}
+	core.Sink(srcs) // want "a source has reached a sink"
+	core.Sink(i1)   // want "a source has reached a sink"
+	core.Sink(i2)   // TODO(#211) want "a source has reached a sink"
+	core.Sink(i3)   // TODO(#211) want "a source has reached a sink"
+	core.Sink(i4)   // TODO(#211) want "a source has reached a sink"
```

Comment:
I maybe misunderstood the intent, since `i4` was previously emitting diagnostic.  Reading #211, the objective would be this current behavior where `i2, i3, i4` are _not_ considered tainted, since that would be outside the scope of this once select.  So these shouldn't be `TODO`s, since they're currently behaving as desired.

Am I misunderstanding #211 here?

If the `select` were within a `for` loop, then they should emit.  (Testing locally, indeed they do.)  That test-case could be added here as well.

Reply:
You're right, this is different from #211.

I've added the `for` loop test case.